.changeset/weak-plums-ring.md

```diff
@@ -0,0 +1,5 @@
+---
+'@chainlink/gsr-adapter': minor
```

Comment:
Should we release this as a major version instead?

Reply:
Yeah, I wouldn't mind, since there are so many framework changes. Will update